zookeeper/source/zk_client_common.cc

```diff
@@ -26,6 +26,7 @@ const char ZkClientCommon::HEALTH_BACKSLASH[] = "/health/";
 const char ZkClientCommon::STATS[] = "/stats";
 const char ZkClientCommon::HEARTBEAT[] = "/heartbeat";
 const char ZkClientCommon::BLOCK_LOCATIONS[] = "/block_locations/";
+const char ZkClientCommon::BLOCK_GROUP_LOCATIONS[] = "/block_group_locations/";
 const char ZkClientCommon::BLOCKS[] = "/blocks";
```

Comment:
should this be "/blocks/"?

Reply:
not something I added. not sure.

Reply:
Based on how it gets concatenated in get_block_metadata_path, it seems like it should be, but if another piece of code expects it to not have the "/" it might break it.

Reply:
I replaced only the places where hardcoded string concatenation "BLOCK_LOCATIONS + SOME_BLOCK_ID" is performed, with my function call which does exactly the original string concatenation for non EC blocks. I don't know where BLOCKS[] is used but my changes should not affect its usage.